libcxx/test/std/iterators/iterator.requirements/iterator.cust/robust_against_ordinary_unqualified_lookup.compile.pass.cpp

```diff
@@ -0,0 +1,28 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Hi sorry, I was not clear previously. Usually we would have one single test robust_against_ordinary_unqualified_lookup.compile.pass.cpp , which contains all the algorithms. so here that single test would test  begin/end/rbegin/rend/iter_move/iter_swap/size, etc...

Reply:
OK, I've merged all three `robust_against_ordinary_unqualified_lookup.compile.pass.cpp` tests into one single `robust_against_poison_pills.compile.pass.cpp` file with additional checks that you've mentioned in the other [comment](https://github.com/llvm/llvm-project/pull/74534#discussion_r1515753847).

I hope current test location `test/std/ranges` is fine. Or maybe it should be put directly in `test/std`?

Commit: https://github.com/llvm/llvm-project/pull/74534/commits/191568d2ffa52273b20cccf4b05cd956bfaf69df

Reply:
Ping @huixie90